### PR TITLE
Add GitHub organization profile

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Adds stub landing to https://github.com/keptn

Signed-off-by: Oleg Nenashev <o.v.nenashev@gmail.com>

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+##  Welcome to the Keptn project 👋
+
+[![Website](https://img.shields.io/static/v1?label=Website&message=keptn.sh&color=blue)](https://keptn.sh/)
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/keptn/projects/1)
+[![Governance](https://img.shields.io/static/v1?label=Governance&message=process&color=yellow)](https://github.com/keptn/keptn/blob/master/GOVERNANCE.md)
+[![Participate](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://keptn.sh/community/)
+
+Keptn is an event-driven orchestration engine for cloud-native apps.
+Started in 2018,
+It uses a declarative approach to build scalable automation
```
Comment (StackScribe) [inline profile/README.md:10]:
```suggestion
it uses a declarative approach to build scalable automation
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+for delivery and operations which can be scaled to a large number of services.
+Keptn evaluates _Service Level Indicators (SLOs)_ and provides a dashboard, alerts, and auto-remediation for them.
```
Comment (StackScribe) [inline profile/README.md]:
```suggestion
Started in 2018,
it uses a declarative approach to automate delivery and operations
so they can be scaled up to a large number of services.
Keptn evaluates _Service Level Indicators (SLOs)_ and provides a dashboard, alerts, and auto-remediation for them.
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+It also provides [integrations](https://keptn.sh/docs/integrations/)
+with the Cloud Native ecosystem 
+including _Prometheus_, _ArgoCD_, _CloudEvents_, _Backstage_,
+and many other cloud-native projects.
+
+Keptn a member project of the [Cloud Native Computing Foundation](https://cncf.io).
```
Comment (StackScribe) [inline profile/README.md]:
```suggestion
Keptn is a member project of the [Cloud Native Computing Foundation](https://cncf.io).
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+**Want to contribute?**
+There is a big community behind Keptn.
+We invite you to join us and participate. 
+Connect to the Community on
```
Comment (StackScribe) [inline profile/README.md]:
```suggestion
Connect to the community on
```
Since "community" is not capitalized in line 21, seems odd to capitalize it here.

**Review by StackScribe — COMMENTED** [review]
lgtm -- just a few editorial suggestions for your consideration.

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**